app/utils/kci_test/__init__.py

```diff
@@ -521,6 +521,9 @@ def import_and_save_test_cases(group_doc_id, group_name, test_cases,
     if not all((group_doc_id, group_name, test_cases)):
         return ret_code
 
+    utils.LOG.info("Importing {} test cases into group {}".format(
```

Comment:
I would use the debug level instead of info so these lines won't appear  in the log when running in production.

Reply:
I actually wanted them to appear in the log, to see how much data is being processed by the server.  I'll put a comment with an example from the server log for a long test suite to see what it looks like.

Reply:
Here's an example with the `v4l2` test plan:
```Dec 05 13:55:08 ip-172-30-0-172 kernelci-backend[8118]: [    INFO/MainThread] 202 POST /callback/lava/test?lab_name=lab-collabora-dev&status=2&status_string=complete (46.235.227.241) 32.45ms
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Received task: lava-test[0a3f1026-58c9-42e0-881c-cafd5b194f59]
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Processing LAVA test data: job 1372968 from lab-collabora-dev
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Generating v4l2 log files in /var/www/images/kernel-ci/gtucker/kernelci-bisect/kernelci-bisect-snapshot-018-3-g7f8476
c6c64f/arm64/defconfig/lab-collabora-dev
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Saving LAVA v2 callback file lava-json-rk3399-gru-kevin.json data in /var/www/images/kernel-ci/gtucker/kernelci-bisec
t/kernelci-bisect-snapshot-018-3-g7f8476c6c64f/arm64/defconfig/lab-collabora-dev
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 5 test cases into group Output-ioctls
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Output-ioctls to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c07998dcf3a0f050590c809
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 4 test cases into group Input/Output-configuration-ioctls
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Input/Output-configuration-ioctls to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c079991cf3a0f050590c821
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 4 test cases into group Streaming-ioctls_Test-input-0
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Streaming-ioctls_Test-input-0 to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c079993cf3a0f050590c831
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 4 test cases into group Allow-for-multiple-opens
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Allow-for-multiple-opens to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c079995cf3a0f050590c83d
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 2 test cases into group Required-ioctls
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Required-ioctls to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c07998dcf3a0f050590c806
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 3 test cases into group Codec-ioctls-Input-0
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Codec-ioctls-Input-0 to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c079990cf3a0f050590c81d
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 6 test cases into group Control-ioctls-Input-0
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Control-ioctls-Input-0 to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c079991cf3a0f050590c826
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 6 test cases into group Input-ioctls
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Input-ioctls to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c079993cf3a0f050590c836
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 2 test cases into group Debug-ioctls
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Debug-ioctls to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c07998ecf3a0f050590c80f
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 10 test cases into group Format-ioctls-Input-0
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Format-ioctls-Input-0 to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c07998fcf3a0f050590c812
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Importing 3 test cases into group Buffer-ioctls-Input-0
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Adding sub-group Buffer-ioctls-Input-0 to v4l2
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Updating test group 'v4l2' (5c07998dcf3a0f050590c805) with sub-group ID 5c079992cf3a0f050590c82d
Dec 05 13:55:08 ip-172-30-0-172 kernelci-celery[8152]: [    INFO/MainThread] Task lava-test[0a3f1026-58c9-42e0-881c-cafd5b194f59] succeeded in 0.245718852151s: [ObjectId('5c07998dcf3a0f050590c80
5')]
```

Reply:
If this is too long, then maybe having one line with the total number of test cases and sub-groups would be better.

Reply:
@ana and now:
```
Dec 05 14:45:41 ip-172-30-0-172 kernelci-backend[12963]: [    INFO/MainThread] 202 POST /callback/lava/test?lab_name=lab-collabora-dev&status=2&status_string=complete (46.235.227.241) 33.44ms
Dec 05 14:45:41 ip-172-30-0-172 kernelci-celery[12997]: [    INFO/MainThread] Received task: lava-test[bed78b8e-eaa9-435a-a8b9-f9c8a978a823]
Dec 05 14:45:41 ip-172-30-0-172 kernelci-celery[12997]: [    INFO/MainThread] Processing LAVA test data: job 1372982 from lab-collabora-dev
Dec 05 14:45:41 ip-172-30-0-172 kernelci-celery[12997]: [    INFO/MainThread] Generating v4l2 log files in /var/www/images/kernel-ci/gtucker/kernelci-bisect/kernelci-bisect-snapshot-018-3-g7f8476c6c64f/arm64/defconfig/lab-collabora-dev
Dec 05 14:45:41 ip-172-30-0-172 kernelci-celery[12997]: [    INFO/MainThread] Saving LAVA v2 callback file lava-json-rk3399-gru-kevin.json data in /var/www/images/kernel-ci/gtucker/kernelci-bisect/kernelci-bisect-snapshot-018-3-g7f8476c6c64f/arm64/defconfig/lab-collabora-dev
Dec 05 14:45:41 ip-172-30-0-172 kernelci-celery[12997]: [    INFO/MainThread] Imported 49 tests in 12 groups from v4l2
Dec 05 14:45:41 ip-172-30-0-172 kernelci-celery[12997]: [    INFO/MainThread] Task lava-test[bed78b8e-eaa9-435a-a8b9-f9c8a978a823] succeeded in 0.224836024921s: [ObjectId('5c07998dcf3a0f050590c805')]
```